packages/react-badge/src/components/Badge/useBadgeStyles.ts

```diff
@@ -9,76 +9,78 @@ export const useRootStyles = makeStyles<BadgeState>([
     null,
     theme => ({
       display: 'inline-flex',
+      boxSizing: 'border-box',
       alignItems: 'center',
       justifyContent: 'center',
       backgroundColor: theme.alias.color.brand.brandBackground,
       borderColor: theme.alias.color.brand.brandBackground,
-      color: theme.global.color.white,
+      color: theme.alias.color.neutral.neutralForegroundInvertedAccessible,
       fontWeight: theme.global.type.fontWeights.semibold,
-      borderWidth: '1px',
+      borderWidth: theme.global.strokeWidth.thin,
       borderStyle: 'solid',
       fontFamily: theme.global.type.fontFamilies.base,
     }),
   ],
   [
     s => s.size === 'smallest',
-    theme => ({
+    {
       width: '6px',
       height: '6px',
       fontSize: '4px',
-    }),
+    },
   ],
   [
     s => s.size === 'smaller',
-    theme => ({
+    {
       width: '10px',
       height: '10px',
       fontSize: '6px',
-    }),
+    },
   ],
   [
     s => s.size === 'small',
-    theme => ({
-      minWidth: '4px',
+    {
+      minWidth: '16px',
       height: '16px',
       paddingRight: '6px',
       paddingLeft: '6px',
       gap: '4px',
       fontSize: '8px',
```

Comment:
Per our conversation, follow up with design to make sure that these font sizes are right, and whether they should be using tokens.

Reply:
I pinged designers, will wait their position on this and open follow up PR